platform/security/handler/security-handler-oidc/src/main/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -15,72 +15,72 @@
  -->
 <metatype:MetaData xmlns:metatype="http://www.osgi.org/xmlns/metatype/v1.2.0">
 
-    <OCD name="OIDC Configuration" id="org.codice.ddf.security.oidc.client.HandlerConfiguration">
+  <OCD name="OIDC Handler Configuration" id="org.codice.ddf.security.oidc.client.HandlerConfiguration">
```

Comment:
📘 Changed spacing, removed unnecessary `required="true"`, and added default values.